cli/types/tslint.json

```diff
@@ -1,14 +1,16 @@
 {
   // settings for linting TypeScript files in this folder
```

Comment:
JSON with comments are weird, and prettier actually removed these comments. I don't think it is a big loss